activation/atxdb.go

```diff
@@ -605,6 +630,24 @@ func (db *DB) GetNodeLastAtxID(nodeID types.NodeID) (types.ATXID, error) {
 	return types.ATXID(types.BytesToHash(nodeAtxsIterator.Value())), nil
 }
 
+// GetEpochAtxs returns all valid ATXs received in the epoch epochID
+func (db *DB) GetEpochAtxs(epochID types.EpochID) (atxs []types.ATXID) {
+	atxIterator := db.atxs.Find(getEpochPrefix(epochID))
+	for atxIterator.Next() {
+		if atxIterator.Key() == nil {
+			break
+		}
+		var a types.ATXID
+		err := types.BytesToInterface(atxIterator.Value(), &a)
+		if err != nil {
+			// log error
+			break
```

Comment:
maybe continue because there might be more valid atxs ? I know this is probably rare bug. at least lets write a log or something

Reply:
maybe this one is a little more important

Reply:
I've added a panic since this should never happen unless db is corrupted